### DISCUSSION TITLE
What format does Any use to store content?

### DISCUSSION THREAD

**Discussion by jehtee** [anyproto/contributors#177] [Q&A] [answered]
We like to future proof our operations using open-source tools and open-standard formats

So I would like to know what file formats does ANY use to store content?  for example, if ANY stores things in markdown file, we could then manage these file in GIT and provide multiple different access paths.  This means some of us could use ANY as the editor while other could just as easily use VSC if that is their preference, for example.

**Comment by filip-anytype** [comment, marked as answer]
Hey @jehtee, at the moment we use our own open-sourced protocol called [Any-block](https://github.com/anyproto/any-block). You can export your spaces and objects into markdown, but there is no option to use markdown (or anything else) instead of Anyblock.
